build-parent/pom.xml

```diff
@@ -328,7 +328,7 @@
           <configuration>
             <!-- ${argLine} placeholder needed due to surefire & jacoco issue -->
             <!-- https://stackoverflow.com/questions/23190107/cannot-use-jacoco-jvm-args-and-surefire-jvm-args-together-in-maven -->
-            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 ${argLine}</argLine>
+            <argLine>-Xmx1024m -Dfile.encoding=UTF-8 @{argLine}</argLine>
```

Comment:
I think we should update also comment? form ${argLine} to @{argLine}?

Reply:
Sure will do it.